elections/ArviZ_2020.md

```diff
@@ -0,0 +1,47 @@
+# Election 2020
+
+## Summary of Process
+This is the first election cycle for ArviZ. During this cycle we are aiming
+for the following outcomes
+
+* Ratification of Governance.MD
+* Identification and listing of Core Contributors
+* Nominations for Council Candidates 
+* Council Elections. Particularly
+  * Size of Council
+  * Inaugural ArviZ Council
+
+
+## Process Details
+### Identification of Core Contributors (1 week)
+1. A Google Form will be created to get core contributor names and emails
+    * Emails are needed for Elections steps so NumFOCUS can mail and track votes
+    * The Google Form is moderated by Ravin
+2. At conclusion of week Ravin will update PR with all names and make the form's responses public,
+with exception of email
+
+
+### Nominations for Council (1 week)
+1. A github issue labeled *Arviz 2020 Council Elections* will be opened for nominations
+2. Anyone can nominate a core contributor (including general public)
```

Comment:
The 1 week term and general public being able to nominate diverges from what is said in governance. We should probably have a couple lines explaining that due to this being the first there will be some exceptions (these are not the only ones, we'll also be voting for number of council members for example)

Reply:
Fair point, let me edit Governance to say something less than no less than 1 week or as not to be too restrictive